scipy/sparse/linalg/isolve/minres.py

```diff
@@ -356,8 +356,11 @@ def minres(A, b, x0=None, shift=0.0, tol=1e-5, maxiter=None,
 
 
 if __name__ == '__main__':
-    from numpy import arange
+    from numpy import arange, zeros, allclose
```

Comment:
All the changes below `__name__ == '__main__'` will only be executed if someone actually runs this `py` file. Hence it won't make any effect on the regular usage. Since SciPy imports the code above it.

Reply:
Hi @ilayn thank you for your review, I'll do the change.

Reply:
Ah I mean that there is no implementation change yet. Because all the changes are invisible to the rest of SciPy since what you added will not be run by importing `minres`.